config/nginx/default

```diff
@@ -0,0 +1,23 @@
+server {
+	listen 80 default_server;
+	listen [::]:80 default_server;
+
+
+	root /var/www/html;
+
+	index index.php;
+
+	server_name _;
+
+	location / {
+		try_files $uri $uri/ =404;
+	}
+
+	location ~ \.php$ {
+		include snippets/fastcgi-php.conf;
+		fastcgi_pass unix:/run/php/php8.2-fpm.sock;
```

Comment:
If php in 8.2 version will be installed on the device, it's ok, but I had to change it when testing on VM :P

Reply:
@esesyn01 FYI, this value should be adjusted to version installed on the image.

Reply:
@punfil Supported php version in meta-oe layer (langdale) is `8.1.16`, so if it generates any trouble, change what is needed

Reply:
Changed PHP version to 8.1.16.